x/blob/test/testdata/block_response.json

```diff
@@ -0,0 +1,1852 @@
+{
```

Comment:
Why did you pick a block with so many transactions?

Reply:
Mostly because this was the one referenced in https://github.com/celestiaorg/celestia-app/issues/2806